packages/libs/src/services/solana/transactionHandler.ts

```diff
@@ -252,6 +254,7 @@ export class TransactionHandler {
           tx.addSignature(new PublicKey(publicKey), signature)
         })
       }
+      txid = bs58.encode(tx.signatures[0]!)
```

Comment:
we okay with the `!`s?

Reply:
There's checks at the beginning of the method to ensure it gets signed